extensions.json

```diff
@@ -587,6 +587,12 @@
       "download": "https://github.com/golang/vscode-go/releases/download/v0.24.1/go-0.24.1.vsix",
       "version": "0.24.1"
     },
+    {
+      "id": "goodbaikin.org-preview",
+      "repository": "https://github.com/goodbaikin/VSC-Org-Preview",
+      "version": "1.0.0",
+      "checkout": "5245d553b65a97a86f5827968918e6a55444cc7d"
```

Comment:
Oops, apologies, in this case it's better to run just:

```
node add-extension https://github.com/goodbaikin/VSC-Org-Preview
```

Explanation: Since the repository was so kind to provide an official `.vsix` GitHub Release, it's much safer to re-publish that directly than to try and repackage the extension ourselves (and risk missing a step, leading to a broken extension package on OpenVSX).

Without a specific `--checkout` argument, `node add-extension <repo>` will check for available GitHub Releases, and automatically use them when available.

Reply:
done - rewrote history this time :)

Reply:
Many thanks!

Huh, the script didn't do what I expected it to do. 🙄 Will look for bugs in it.

Reply:
Ah, found the culprit: The script didn't pick `'/releases/download/1.0.0/org-preview-1.0.0.vsix'` [because of the `-pre` in it](https://github.com/open-vsx/publish-extensions/blob/ebb7721d9d1dd8bf1182e8ccbe83de5a178b2c42/lib/github-scraper.js#L10) (thinking it's a pre-release 😅).

Reply:
@KlavsKlavsen I've fixed this bug in https://github.com/open-vsx/publish-extensions/commit/645a1ed5abe0be20f1f789f1ee28e024b87941d5 -- could you please rebase and re-run the script? 🙏

Reply:
merged latest master and re-run :)

Reply:
Ah... 😁 perfection! Many thanks 🙏